packages/web-components/src/anchor/anchor.styles.ts

```diff
@@ -1,32 +1,28 @@
-import { css, ElementStyles } from '@microsoft/fast-element';
+import { ElementStyles } from '@microsoft/fast-element';
 import { ElementDefinitionContext, FoundationElementDefinition } from '@microsoft/fast-foundation';
 import {
-  AccentButtonStyles,
-  baseButtonStyles,
-  HypertextStyles,
-  LightweightButtonStyles,
-  OutlineButtonStyles,
-  StealthButtonStyles,
+  _accentButtonStyles,
+  _baseButtonStyles,
+  _hypertextStyles,
+  _lightweightButtonStyles,
+  _neutralButtonStyles,
+  _outlineButtonStyles,
+  _stealthButtonStyles,
```

Comment:
This is strange since we don't have this naming convention anywhere else in any of our codebases.

Reply:
Fair. I was under the impression that internals weren't considered part of the public API and I was just resolving the warning the API report generates. I've reverted this.

Reply:
I don't think you were wrong in doing that. It's more of what we have as a patter. I think we could talk about doing this more broadly in the future.